floss/utils.py

```diff
@@ -10,6 +10,7 @@
 from typing import Set, Tuple, Iterable, Optional
 from pathlib import Path
 from collections import OrderedDict
+import sys
```

Comment:
please run this through isort, black, etc. so the code styler linter passes

Reply:
Sorry, I didn't notice the code linter.😂 Should have fixed it in the latest commit.